lib/runner/extensions/event.command.js

```diff
@@ -1,4 +1,6 @@
 var _ = require('lodash'),
+    async = require("async"),
+
```

Comment:
I like to keep internal and external requires separated by an empty line :smile: